src/plugins/wizard/public/visualizations/pie_chart/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { VisualizationTypeOptions } from '../../services/type_service/visualization_type';
+
+export const createPieChartConfig = (): VisualizationTypeOptions => {
+  return {
+    name: 'pie_chart',
```

Comment:
No description required?

Reply:
This is still WIP. So ive just put a placeholder here for now to activate the type picker and make sure that it works